### PR TITLE
i64 marshal/unmarshal to Timestamp/Long

### PR DESCRIPTION
**Pull request by oliviazhang0809**

### CODE REVIEW

File: examples/example-gateway/build/gen-code/clients/contacts/contacts/types_i64.go [hunk 1]
```diff
@@ -0,0 +1 @@
+package contacts
```
Comment (Raynos) [inline examples/example-gateway/build/gen-code/clients/contacts/contacts/types_i64.go]: It looks like we generate unneeded empty `types_i64.go` files. We want to somehow not do that.

File: scripts/resolve_i64/main.go [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+	sort.Sort(I64Structs(meta.Types))
+	tmpl, err := template.ParseFiles(filepath.Join(getDirName(), templateFileName))
+	if err != nil {
+		fmt.Fprintln(os.Stderr, fmt.Errorf(
+			"error parsing template %s: %s", templateFileName, err.Error()))
+		os.Exit(1)
+		return
+	}
+	tplBuffer := bytes.NewBuffer(nil)
+	err = tmpl.Execute(tplBuffer, meta)
```
Comment (Raynos) [inline scripts/resolve_i64/main.go]: if `len(meta.Types) == 0` we can just end here and not do any templating at all.

File: scripts/resolve_i64/main.go [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+func main() {
+	thriftFile := os.Args[1]
+	module, err := compile.Compile(thriftFile)
+	if err != nil {
+		panic(fmt.Sprintf("Failed to parse thrift file: %s", thriftFile))
+	}
+	meta := &Meta{}
+
+	s := strings.TrimSuffix(thriftFile, ".thrift")
+	s = strings.Replace(s, "/idl/", "/build/gen-code/", 1)
```
Comment (Raynos) [inline scripts/resolve_i64/main.go]: we can leave a TODO here. `/build/gen-code/` is hardcoded and we might want to customize that; maybe via `os.Args[2]`

File: examples/example-gateway/build/gen-code/clients/contacts/contacts/types_i64.go (hunk 1 above)
Comment (oliviazhang0809) [reply examples/example-gateway/build/gen-code/clients/contacts/contacts/types_i64.go]: sounds good

**Pull request merged**